scoring/test/cscore-test.metta

```diff
@@ -28,10 +28,11 @@
                     (Cons (Cons False (Cons False (Cons False Nil))) Nil))))
                     (Cons A (Cons B (Cons Output Nil)))))
 
-! (assertEqual (getCscore ttable1 (buildTree (AND A B)) 2.0) (mkCscore -1 2 1 0 -2))
-! (assertEqual (getCscore ttable1 (buildTree (OR A B)) 4) (mkCscore 0 2 0.5 0 -0.5))
-! (assertEqual (getCscore ttable2 (buildTree (AND A B)) 2.0) (mkCscore 0 2 1.0 0 -1.0))
-! (assertEqual (getCscore ttable2 (buildTree (OR A B)) 1) (mkCscore -2 2 2.0 0 -4.0))
+! (assertEqual (getCscore ttable1 (buildTree (AND A B)) 2.0 &cacheSpace) (mkCscore -1 2 1 0 -2))
```

Comment:
@eyuuab, Just a quick question, where is the `&cacheSpace` coming from in this file?